payments-core/src/main/java/com/stripe/android/payments/bankaccount/ui/CollectBankAccountViewModel.kt

```diff
@@ -272,3 +278,7 @@ internal class CollectBankAccountViewModel @Inject constructor(
         private const val KEY_HAS_LAUNCHED = "key_has_launched"
     }
 }
+
+private fun CollectBankAccountConfiguration.retrieveElementsContext(): ElementsContext? {
+    return (this as? InstantDebits)?.elementsContext
+}
```

Comment:
same as above : )

Reply:
This one is a little trickier, since `CollectBankAccountConfiguration.USBankAccount` is public. Good if we revisit this later this year for the next major version bump?

Reply:
sounds good! When we do that ideally we also move it out of the configuration object (that intends to be public facing) and follow the Args pattern we do on FC 😄